### DISCUSSION TITLE
pipe expressions

### DISCUSSION THREAD

**Discussion by mmeqbel-wiley** [prettier/prettier#13958] [Q&A]
### how can we print each pipe body in a new line?

**input**:
```
export default Component
	|> withCss 
	|> withMemo;
```

**expected**: 

```
export default Component
	|> withCss 
	|> withMemo;
```

**actual**: 

`export default Component |> withCss |> withMemo;`

**Comment by kachkaev** [comment]
You can add `//` at the end of the first line:

**Prettier 2.8.1**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEcAeAHCAnGACAEzgDMBDAVwBt8BhCAWyygXwHpWAdKDmAHwD48AdwCWMABY0AzlLxceA4WPEBZOPQgBuLiAA0ICBhgjoU5KFLZsEIQAVLCMylKUhpAJ5n9AI2ykwANZwMADKpPRwADIizMhklFJwPn6BwSEY-jEA5sgw2ORJqPTecAREBJGkUFnkpFlwAGI49KQwxtXIIBQwEHog4jD0lADq4mJwUhlgcCGOYiIAbmLunWAyfTGJuLZ+WS1xLon6AFZSaCHZlHAAiuQQ8AcJhRnYW53epCWUfRjYMTDDEQECTIAAcAAZ9L8IIlhn4MJ1fhM4NgFnA+gBHO7wHaGJxdKQAWmYpVKfWwcCxIgpOzq+yQ8SOIES9BEuXyhSklzgAEE2n9vOQcSjorEGYdOdzbvd0eKnvoYJ9AcDxMgAEwKvwiSjZOj0emoKQAVj65ESABVPk5GYUFgUAJJQIiwEJgP5GHlOkIwdxXR6JAC+AaAA)

```sh
--parser babel
```

**Input:**

```jsx
export default Component //
	|> withCss 
	|> withMemo;

```

**Output:**

```jsx
export default Component //
  |> withCss
  |> withMemo;

```
